Gemfile

```diff
@@ -38,12 +38,16 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'devise'
 gem 'will_paginate', '~> 3.1.0'
 gem 'rails-html-sanitizer'
-gem 'letsencrypt_plugin'
+#gem 'letsencrypt_plugin'
 
 gem 'whenever', :require => false
 
 gem 'coveralls', require: false
 
+# Gems for the project Mentor 
+
+gem 'stripe'
```

Comment:
@schas002 @ArtOfCode-  I want to give you more info about this. There will be 2 type of users. Mentors will be able to earn money by helping other users solve their coding issues, Students will be paying a 20-50$ subscription to get help.

Payment can be also done in Bitcoin. 

I know that you don't like this aspect, but some developers may not be able to provide help for free, because they need to have a wage, on the otherside many students are learning ROR, but are challenged and need some directions..

I read your license, I hope this does not disturb you too much.. The project is open source. Other developers will be able to fork this, contribute and deploy their own instance.

Thanks 
Fabrizio